src/rgw/rgw_cache.h

```diff
@@ -333,467 +355,10 @@ class ObjectCache {
   void set_enabled(bool status);
 
   void chain_cache(RGWChainedCache *cache);
+  void unchain_cache(RGWChainedCache *cache);
   void invalidate_all();
 };
 
-template <class T>
```

Comment:
um, I must have missed this (due to size in github ui) in the first, first datacache commit;  but why is it moving?

Reply:
probably merge conflict resolution artifact, eventually moved below to:
https://github.com/ceph/ceph/blob/220acce1b33a677c6a4a9609f15cc974ba4242a7/src/rgw/rgw_cache.h#L366